src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -2668,5 +2670,158 @@ class C
 }";
             await TestAsync(code, expected, index: 2, compareTokens: false);
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable), Trait(Traits.Feature, Test.Utilities.Traits.Features.Tuples)]
+        public async Task Tuples_Disabled()
+        {
+            var code =
+@"class C
+{
+    var i = [|(1, ""hello"")|].ToString();
+}";
```

Comment:
what happens if i select individual items that form this tuple and apply the same refactoring. does it generate valid syntax? or should that be an invalid selection and the refactoring not be offered? 
E.g: Select `1` and invoke Introduce variable.